src/SocialServer/Database/ChatGroup.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Melia.Social.Database
+{
+	/// <summary>
+	/// Represents a chat group.
```

Comment:
What does that mean though?^^

Reply:
![image](https://github.com/NoCode-NoLife/melia/assets/3139467/d46d3b97-ba60-4e61-9d2d-6cb62484f8bf)
Whatever these are xD?

Reply:
So... chat rooms between two players? Potentially more?